server/lib/settings.ts

```diff
@@ -182,6 +182,12 @@ interface NotificationSettings {
   agents: NotificationAgents;
 }
 
+interface JobSettings {
+  schedule: string;
+}
+
+type JobName = string;
```

Comment:
I don't really like having the job name as a string. It in some ways defeats the purpose of using a Record. We should have a type of all the possible job ids (which shouldn't be changed from the IDs they were before) and use that as the type on the Record.

```javascript
type JobIds = 'plex-recently-added' | 'some-other-job';

Record<JobIds, JobSettings>
```

Then later, typescript will know which jobs we can reference and we avoid possible undefined objects.